apps/explorer/src/ui/CoinBalance.tsx

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Heading } from '~/ui/Heading';
+import { Text } from '~/ui/Text';
+
+// Passing amount as a string or number for optional number suffix
+export type CoinBalanceProps = { amount: number | string; symbol?: string };
+
+export function CoinBalance({ amount, symbol }: CoinBalanceProps) {
+    return (
+        <div className="flex flex-row items-end gap-1 text-sui-grey-100 break-words">
```

Comment:
No need to change this but FWIW `flex-row` is the default.

Reply:
got it